squire_lib/src/tournament.rs

```diff
@@ -353,6 +354,7 @@ impl Tournament {
     /// Removes players from the tournament that did not complete registration.
     /// This include players that did not submit enough decks (defined by `require_deck_reg` and
     /// `min_deck_count`) and that didn't check in (defined by `require_check_in`).
+    #[no_panic]
```

Comment:
Does `no_panic` not apply transitively? If not, that's somewhat unfortunate.

Reply:
It does apply transitively, which is why I can't add it to `apply_op` just yet.